src/operator/src/insert.rs

```diff
@@ -320,7 +320,7 @@ impl Inserter {
                 Ok(())
             }
             Err(err) => {
-                error!(
+                debug!(
```

Comment:
I don't think it's a good idea to change this log from error to debug. We may miss the critical log.

Reply:
<img width="1324" alt="image" src="https://github.com/GreptimeTeam/greptimedb/assets/990479/99e58b7d-87a1-4f07-9eb7-4faeb3ef45d7">

It's rather boring to see the logs like this. We won't miss the critical logs because all the errors will be dealt in user interface layer (and log there).

Reply:
I think it's the user's responsibility to resolve this error.

If we only print the error in the upper layer, maybe we lose some location or other critical info.

Reply:
It's of course user's responsibility to resolve the error, just don't log meaningless error in db. The upper layer has full error stack trace if the error is supposed to log. But if you say so.